src/components/ReviewCard.jsx

```diff
@@ -1,10 +1,10 @@
 export default function ReviewCard({title, owner, votes,}) {
     return(
         <article className="ReviewBox">
-            <h3 className="ReviewOwner">{owner}</h3>
             <div className="ReviewBoxDetails">
-            <h4 className="ReviewTitle">{title}</h4>
             <p className="ReviewVotes">{votes}</p>
+            <h3 className="ReviewOwner">{owner}</h3>
+            <h4 className="ReviewTitle">Title:{title}</h4>
```

Comment:
I would expect this to be the main element on the page but is currently the smallest element, normally the title is at the top and is in large font.

Reply:
ye i was having issue with flex, it was jumping around and i was playing with them until they got to perfect position